docs/real-world-examples/ccfraud.md

```diff
@@ -31,72 +31,21 @@ Alternatively, you can just install the required dependencies:
 python -m pip install -r ./examples/pipelines/requirements.txt
 ```
 
-## Provision an FL sandbox workspace
+## Provision an FL sandbox (with your kaggle credentials!)
 
-To run this example, you will need to provision an AzureML workspace ready for Federated Learning. We strongly recommend you use the setup provided in the repository [quickstart](../quickstart.md). We will use the same names for the computes and datastores created by default during this quickstart.
+To run this example, you will need to provision one of our [sandboxes](../provisioning/sandboxes.md). Any sandbox should work with this tutorial below (if not, please [reach out](https://github.com/Azure-Samples/azure-ml-federated-learning/issues)). We will use the same names for the computes and datastores created by default in those sandboxes.
 
-:notebook: take note of your workspace name, resource group and subscription id. You will need them to submit the experiment.
-
-## Add your Kaggle credentials to the workspace key vault
-
-In the next section, we will run a job in the AzureML workspace that will unpack the demo dataset from Kaggle into each of your silos.
-
-Kaggle requires a username and an [API key](https://github.com/Kaggle/kaggle-api#api-credentials), so we will first store safely those credentials in the workspace key vault.
-
-### Option 1: using Azure CLI
-
-1. Let's first obtain your AAD identifier (object id) by running the following command. We'll use it in the next step.
-
-    ```bash
-    az ad signed-in-user show --query id
-    ```
-
-2. Create a new key vault policy for yourself, and grant permissions to list, set & delete secrets.
-
-    ```bash
-    az keyvault set-policy -n <key-vault-name> --secret-permissions list set delete --object-id <object-id>
-    ```
-
-    > Note: The AML workspace you created with the aforementioned script contains the name of the key vault. Default is `ws-shkv-fldemo`.
-
-3. With your newly created permissions, you can now create a secret to store the `kaggleusername`.
-
-    ```bash
-    az keyvault secret set --name kaggleusername --vault-name <key-vault-name> --value <kaggle-username>
-    ```
+If you have already provisioned a sandbox during the [quickstart](../quickstart.md) you can reuse it, but you need to **make sure you have added your [Kaggle](https://www.kaggle.com/) credentials** so we can upload the required dataset in each silo. If not, please refer to our [tutorial on how to add your kaggle credentials](../tutorials/add-kaggle-credentials.md) to the workspace secret store before running the following sections.
 
-    > Make sure to provide your *Kaggle Username*.
+You can also [re-provision another sandbox](../provisioning/sandboxes.md) with a different base name using the deploy buttons above, and provide your **kaggleUsername** and **kaggleKey** as parameters, so they will be injected securely in your workspace secret store.
```

Comment:
minor: not "above", "on the sandboxes page" or something like that

Reply:
fixed!